src/index.html

```diff
@@ -6,8 +6,39 @@
     <title>Moramia</title>
 </head>
 <body>
-    <h1>Hello Push</h1>
-    <img src="./assets/img/moramia.png" alt="image2">
-    <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ea rem, placeat assumenda facere voluptatem amet molestias tempora reiciendis quibusdam non, eaque dolorem natus labore ut, magni facilis itaque! Blanditiis, error?</p>
+   <main class="products-wrapper">
+        <article class="product">
+            <img src="./assets/img/product1.jpg" alt="Frutos secos">
+            <div class="product-content">
+                <div class="line"></div>
```

Comment:
use a pseudo element to achieve this, empty divs are not good for semantics, if it's a visual decoration it's better to do it using `::after` or `::before`